seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/JsonUtils.java

```diff
@@ -59,6 +59,8 @@ public class JsonUtils {
                     .configure(REQUIRE_SETTERS_FOR_GETTERS, true)
                     .setTimeZone(TimeZone.getDefault());
 
+    private static final ObjectMapper DEFAULT_OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Why we don't directly use `OBJECT_MAPPER` here?

Reply:
Of course, it might be possible to use `OBJECT_MAPPER`. I found that `OBJECT_MAPPER` uses some different configurations than the default. Without knowing the intent of these configurations, it's safer for me to use the default `OBJECT_MAPPER` here.